src/components/Dashboard/AccountBanner.js

```diff
@@ -187,61 +190,48 @@ const BannerWithProbability = ({ activeBalance }) => {
         align-items: center;
       `}
     >
-      {(() => {
-        if (probablilityTooLow) {
-          return (
-            <span css="margin-right: 8px">
-              Activate more ANJ to increase your chances of being selected as a
-              juror
-            </span>
-          )
-        }
-        return (
-          <>
-            <span css="margin-right: 8px">
-              On average, you will be drafted
-              <span
-                css={`
-                  color: ${theme.accent};
-                `}
-              >
-                {` 1 on every ${chances} times `}
-              </span>
-            </span>
-            <Help hint="How is the probability calculated?">
-              Probability of being drafted is calculated dividing your active
-              ANJ balance by the total Court active ANJ, during the current
-              term.
-            </Help>
-          </>
-        )
-      })()}
+      <span
+        css={`
+          margin-right: ${1 * GU}px;
+        `}
+      >
+        {'On average, you will be drafted into a jury '}
+        <span
+          css={`
+            color: ${theme.accent};
+          `}
+        >
+          1 in {chances} times
+        </span>
+      </span>
+      <Help hint="How is the probability calculated?">
+        <p>
+          This is a numerical estimate of your likelihood of being selected for
+          arbitration. It’s calculated by dividing your active ANJ balance
+          against the Court's total active ANJ balance during the current term.
+        </p>
+        <p
+          css={`
+            margin-top: ${1 * GU}px;
+          `}
+        >
+          You currently have &lt;1% of all activated ANJ and hence are unlikely
```

Comment:
I think we should only be showing this if the percentage is actually <1%, right?